frozendict/FrozenDict.py

```diff
@@ -206,4 +196,33 @@ def serialize(self) -> Mapping:
                 for k, v in self.items()}
 
 
+def no_copy_frozendict(value: Mapping[KT, VT_co] = None,
```

Comment:
The problem with this approach is that we can't do the following:

`from frozendict import no_copy_frozendict as FrozenDict
fd = FrozenDict({'k_1': 1.0})
assert isinstance(fd, FrozenDict)`

Reply:
For me this pass and is acceptable:
`def test_imutable_no_copy_1():
    from frozendict import no_copy_frozendict as frozendict
    d = {'k_1': 0, 'k_2': 1}
    fd_2: FrozenDict[str, int] = frozendict(d)
    assert isinstance(fd_2, FrozenDict)
    assert fd_2.items() == d.items()
    d['k_3'] = 3
    assert d.get('k_3') == 3
    assert fd_2.get('k_3') == 3

    fd_3: FrozenDict[str, Dict[str, int]] = frozendict(k=fd_2)
    assert isinstance(fd_3, FrozenDict)
    assert d.get('k_3') == 3
    assert fd_3.get('k').get('k_3') == 3`

Reply:
But well if you need that explicitly I can fix it